frontend/src/components/FormReview/styles.scss

```diff
@@ -23,7 +23,7 @@
   padding-left: 1rem;
 }
 
-.description-section {
+.item-description-section {
   @include type.type-style('body-compact-01');
   color: var(--#{vars.$bcgov-prefix}-text-secondary);
   padding-top: 0.25rem;
```

Comment:
I modified a `<p>`'s class name in the TitleAccordion component, so this is just updating the class name since it's used here as well
![image](https://user-images.githubusercontent.com/25162561/227384390-19601c17-c9d8-402a-8f31-28ecae028a17.png)